### DISCUSSION TITLE
Show if a PR was merged using administrator rights

### DISCUSSION THREAD

**Discussion by Kiemes** [community/community#37303] [Pull Requests]
Under some circumstances, administrators can merge PRs even when PRs require a review and none was done.
Using the administrator rights have to be confirmed when merging the PR.

![Screenshot 2022-10-26 at 09 08 19](https://user-images.githubusercontent.com/13349924/197958192-934c7ce9-4867-4502-b662-22fe36f24ea1.png)

For full audit compliance, a PR should show that admin rights were used to merge the PR.
At the moment we only see `<user> merged commit 1234567 into master x days ago`.
Our suggestion would be to have `<user> merged commit 1234567 into master x days ago using administrator right` or something similar.

**Comment by ossanna16** [comment]
Hi there @Kiemes  and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
